5a/readme.md

```diff
@@ -3,20 +3,29 @@
 This solution builds on top of solution 4 and adds F# as an option for potential library code. We often find ourselves in a situation where we'd like to use F# "business logic" interfaced by C# orleans code. This repository illustrates how to do just that using the `Library` project.
 
 ## running the code
+
 ### silos
+
 As in example 4, following scripts can be used to run silos:
-* `./run-silo-docker.sh`
-* `./run-silo-local.sh`
-  Important thing to note is that if one wants to run multiple silos which form a cluster, following variables need to be made unique for every silo:
-* `GATEWAYPORT` (e.g. `3001`, `3002`, etc.)
-* `SILOPORT` (e.g. `2001`, `2002`, etc.)
-* `DASHBOARDPORT` (e.g. `8081`, `8082`, etc.)
+
+- `./run-silo-docker.sh`
+- `./run-silo-local.sh`
+  An important thing to note is that if one wants to run multiple silos which form a cluster, the following variables need to be made unique for every silo:
+- `GATEWAYPORT` (e.g. `3001`, `3002`, etc.)
+- `SILOPORT` (e.g. `2001`, `2002`, etc.)
+- `DASHBOARDPORT` (e.g. `8081`, `8082`, etc.)
```

Comment:
What, why would you propose such a radical change??? But seriously, why though 😅

Reply:
Just looking into the file for possible changes in wording and I saw a few mistakes. Fixed and the markdown autoformatting did the rest. 🙈

Reply:
That's fine. All markdown files use `*` for bullet points. It's fine, it really doesn't matter.